sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -126,12 +127,22 @@ querySpec
     | VALUES values (',' values)*                                                    #valuesRelation
     ;
 
+
 selectItem
-    : expr (AS? ident)?                                                              #selectSingle
+    : selectSingleItem                                                               #defaultSelectSingle
     | qname '.' ASTERISK                                                             #selectAll
     | ASTERISK                                                                       #selectAll
     ;
 
+selectSingleItem
+    : expr (AS? ident)?
+    ;
+
+returning
+    : RETURNING selectSingleItem (',' selectSingleItem)*
+    | RETURNING ASTERISK
+    ;
```

Comment:
PostgreSQL also supports the `qname '.' ASTERISK` notation. So I think it would actually make sense to simply re-use the full `selectItem`.

```
update tbl set x = x + 1 returning tbl.*, x, *
;
 x | x | x 
---+---+---
 4 | 4 | 4
(1 row)
```

Reply:
👍